main_test.go

```diff
@@ -96,19 +96,16 @@ func TestGoPyErrors(t *testing.T) {
 	if err != nil {
 		t.Fatalf("could not create workdir: %v\n", err)
 	}
-	fmt.Printf("pyvm: %s making work dir: %s\n", pyvm, workdir)
-	//defer os.RemoveAll(workdir)
+	t.Logf("pyvm: %s making work dir: %s\n", pyvm, workdir)
+	defer os.RemoveAll(workdir)
 
 	curPkgPath := reflect.TypeOf(pkg{}).PkgPath()
-
-	// fmt.Printf("building in work dir: %s\n", workdir)
 	fpath := filepath.Join(curPkgPath, "_examples/gopyerrors")
-	args := []string{"run", ".", "gen", "-vm=" + pyvm, "-output=" + workdir, fpath}
-	cmd := exec.Command("go", args...)
-	fmt.Printf("running: %v\n", args)
+	cmd := exec.Command("go", "run", ".", "gen", "-vm="+pyvm, "-output="+workdir, fpath)
+	t.Logf("running: %v\n", cmd.Args)
 	out, err := cmd.CombinedOutput()
 	if err != nil {
-		t.Fatalf("cmd: go %v %v\n", strings.Join(args, " "), err)
+		t.Fatalf("could not run %v %v\n", strings.Join(cmd.Args, " "), err)
```

Comment:
```suggestion
		t.Fatalf("could not run %v: %+v\n", strings.Join(cmd.Args, " "), err)
```
(to leverage the new error-wrapping of Go-1.13. one day)

Reply:
done!